### DISCUSSION TITLE
Nicegui documentation as PDF

### DISCUSSION THREAD

**Discussion by alikalik9** [zauberzeug/nicegui#1571] [Ideas / Feature Requests]
Is there a PDF of the whole documentation? I would like to try using it with Chatgpt/Claude 2 combined with the Chatgpt prompt from the wiki. For a Python/Webdev beginners like me ChatGPT ist incredibly helpful and it would be awesome if we could give ChatGPT profound knowledge about Nicegui.

**Comment by rodja** [comment]
Thanks for asking @alikalik9. We do not have a pdf but there are some interesting approaches to feed LLMs with informations about NiceGUI mentioned in #1371. I will close this as a duplicate. Feel free to continue the conversion over there.
